### ISSUE TITLE
Opening WebGL page throws  “THREE.WebGLRenderer: Error creating WebGL context”

### ISSUE THREAD

**Issue by tank2737** [seleniumbase/SeleniumBase#1873]
When using only selenium, I did not encounter this problem.how should I handle this？

**Comment by mdmintz** [comment]
Add this command-line option: ``--enable-3d-apis``

Eg. ``pytest --enable-3d-apis``

**Comment by tank2737** [comment]
Recorder Mode INVALID OPTION

**Comment by stefanoantonel** [comment]
I'm having this issue again.

Script looks:

```
# webgl.py
from seleniumbase import BaseCase
import time
BaseCase.main(__name__, __file__)


class MyTestClass(BaseCase):
    def test_swag_labs(self):
        self.open("https://threejs.org/examples/#webgl_interactive_lines")
        time.sleep(60)
```

Command line:
`venv/bin/pytest webgl.py --enable-3d-apis`

Chrome error:
```
...
three.module.js:29124 THREE.WebGLRenderer: A WebGL context could not be created. Reason:  Could not create a WebGL context, VENDOR = 0xffff, DEVICE = 0xffff, GL_VENDOR = Google Inc. (Google), GL_RENDERER = ANGLE (Google, Vulkan 1.3.0 (SwiftShader Device (LLVM 10.0.0) (0x0000C0DE)), SwiftShader driver-5.0.0), GL_VERSION = 5.0.0, Sandboxed = no, Optimus = no, AMD switchable = no, Reset notification strategy = 0x8252, ErrorMessage = BindToCurrentSequence failed: .
...
three.module.js:28700 THREE.WebGLRenderer: Error creating WebGL context.
```

Even with `--swiftshader` flag I get the same issue

Chrome version: 
Version 121.0.6167.85 (Official Build) (arm64)
Running on M1 Max

**Comment by mdmintz** [comment]
@stefanoantonel This is now fixed in `4.23.1` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.23.1

Use `--enable-3d-apis` to activate it from `pytest`, or `enable_3d_apis=True` from the `SB()` format.

**Comment by stefanoantonel** [comment]
@mdmintz Thank you so much for this quick fix! 
It is working like a charm :)
